### PR TITLE
Swap out the telemetry helpers for the telemetry package

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
removes all the telemetry code so that this codemod and no-implicit-this can use the same telemetry code

### CODE REVIEW

File: test/fixtures/input/app/app.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 import loadInitializers from 'ember-load-initializers';
 import config from './config/environment';
 
-const App = Application.extend({
-  modulePrefix: config.modulePrefix,
-  podModulePrefix: config.podModulePrefix,
-  Resolver,
-});
+@classic
+class AppApplication extends Application {
```
Comment (rajasegar) [inline test/fixtures/input/app/app.js]: Do we really need the name change here, can we just keep it App

File: test/fixtures/input/app/router.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
+import classic from 'ember-classic-decorator';
 import EmberRouter from '@ember/routing/router';
 import config from './config/environment';
 
-const Router = EmberRouter.extend({
-  location: config.locationType,
-  rootURL: config.rootURL,
-});
+@classic
+class RouterRouter extends EmberRouter {
```
Comment (rajasegar) [inline test/fixtures/input/app/router.js]: Same here, RouterRouter doesn't sound nice to me.

File: test/fixtures/input/app/app.js (hunk 1 above)
Comment (NullVoxPopuli) [reply test/fixtures/input/app/app.js]: I should not have committed the integration test results. Whoops

Comment (NullVoxPopuli) [reply test/fixtures/input/app/app.js]: fixed

File: test/fixtures/input/app/router.js (hunk 2 above)
Comment (NullVoxPopuli) [reply test/fixtures/input/app/router.js]: reverted

**Pull request merged**